Split/FetcherEngine/Recorder/EventsRecorderWorker.swift

```diff
@@ -46,5 +49,10 @@ class EventsRecorderWorker: RecorderWorker {
         } while rowCount == eventsPerPush
         // Activate non sent events to retry in next iteration
         eventsStorage.setActive(failedEvents)
+        eventsStorage.setActive(failedEvents)
```

Comment:
duplicated sentence? (lines 51 and 52 are the same)

Reply:
Yes! Good catch doc!